src/main/java/io/openliberty/tools/intellij/runConfiguration/LibertyRunManagerListener.java

```diff
@@ -40,8 +41,11 @@ public void runConfigurationRemoved(@NotNull RunnerAndConfigurationSettings sett
             try {
                 VirtualFile vBuildFile = VfsUtil.findFile(Paths.get(runConfig.getBuildFile()), true);
                 LibertyModule libertyModule = libertyModules.getLibertyModule(vBuildFile);
-                if (libertyModule != null && libertyModule.getCustomStartParams().equals(runConfig.getParams())) {
-                    libertyModule.setCustomStartParams(""); // clear the custom params
+                if (libertyModule != null) {
+                    libertyModule.setRunInContainer(false); // clear the run in container checkbox
```

Comment:
Here we can see that every time `setCustomStartParams()` is called we also need to call `setRunInContainer()`. This is done in `LibertyRunConfiguration`. Could you open an issue to redesign this interface so that the params and the boolean must be set in one call? Now may not be a good time to redesign this interface.

Reply:
@turkeylurkey This means we should have a single method for eg: setParamsAndRunInContainer(String params, Boolean runInContainer)  in LibertyRunConfiguration ?
If we implement a single method like this, when a user checks the 'Run In Container' option, we also need to pass parameters along with the action of enabling or disabling the checkbox, correct?

Reply:
Yes, but we will have the Run configuration open in the UI when the checkbox is used. This is part of #977.